### PR TITLE
fixed faulty get visitor query

### PR DESCRIPTION
**Pull request by gabrielsaruhashi**
image

### CODE REVIEW

File: src/graphql/GetVisitor.graphql [hunk 1]
```diff
@@ -1,7 +1,7 @@
 query GetVisitor($visitorId: ID!, $facilityId: ID!) {
   visitor(id: $visitorId) {
     ...VisitorFragment
-    connections(facilityId : $facilityId}) {
+    connections(facilityId: $facilityId) {
```
Comment (gabrielsaruhashi) [inline src/graphql/GetVisitor.graphql:4]: i'm surprised there was a typo here in prod
was driving me crazy trying to figure out why it was not updating the .generated ts query

File: src/graphql/GetVisitor.graphql [hunk 2]
```diff
@@ -50,7 +50,7 @@ query GetVisitor($visitorId: ID!, $facilityId: ID!) {
             name
             description
             enabled
-            group {
+            groups {
```
Comment (wittejm) [inline src/graphql/GetVisitor.graphql:53]: Groups has an id? Isn't it an array?

**Review by wittejm — APPROVED** [review]

File: src/graphql/GetVisitor.graphql (hunk 2 above)
Comment (gabrielsaruhashi) [reply src/graphql/GetVisitor.graphql:53]: hm? this is the syntax for it regardless of whether or not is an array. what matters is this ent
https://graphql.org/learn/queries/

**Review by crcommons — APPROVED** [review]

File: src/graphql/GetVisitor.graphql (hunk 1 above)
Comment (cainlevy) [reply src/graphql/GetVisitor.graphql:4]: this must be why i didn't discover the bug before changing the schema 😭

File: src/graphql/GetVisitor.graphql (hunk 2 above)
Comment (cainlevy) [reply src/graphql/GetVisitor.graphql:53]: we can remove `groups { id }`. it's unused.

**Pull request merged**